app/views/unattended/report_templates/subscription_-_general_report.erb

```diff
@@ -37,7 +37,8 @@ require:
           'Start date': pool.start_date,
           'End date': pool.end_date,
           'Days Remaining': pool.days_until_expiration,
-          'Organization': pool.organization.name
+          'Organization': pool.organization.name,
+          'Product Host Count': pool.product_host_count
         ) -%>
 <%- end -%>
-<%= report_render -%>
\ No newline at end of file
+<%= report_render -%>
```

Comment:
No need to add a newline here, but if lint is happy I'm happy

Reply:
Not sure what is happening.